src/scss/root/_index.scss

```diff
@@ -0,0 +1,112 @@
+@use 'sass:list';
+@use 'sass:map';
+
+@forward 'reset';
+@forward 'a11y';
+
+$alias: () !default;
+$breakpoints: () !default;
+$global: () !default;
+$tokens: () !default;
+
+@mixin block-components($blockName) {
+	@if list.length(map.get($tokens, 'default', 'blocks', $blockName, 'components')) >= 0 {
+		@each $name, $value in map.get($tokens, 'default', 'blocks', $blockName, 'components') {
+			@if type-of($value) == map {
+				@each $a, $b in $value {
+					--c-#{$name}-#{$a}: #{$b};
+				}
+			}
+		}
+	}
+}
+
+@mixin block-properties($blockName) {
+	@each $name, $value in map.get($tokens, 'default', 'blocks', $blockName) {
+		@if map.has-key($tokens, 'default', 'blocks', $blockName) {
+			@if type-of($value) != map {
+				#{$name}: block-var($blockName, $name);
+			}
+		}
+	}
+}
+
+@mixin component-properties($component) {
+	@if map.has-key($tokens, 'default', 'components', $component) {
+		@each $name, $value in map.get($tokens, 'default', 'components', $component) {
+			#{$name}: component-var($component, $name);
+		}
+	}
+}
+
+@function component-var($component, $item) {
+	@return var(--c-#{$component}-#{$item});
+}
+
+@function block-var($block, $item) {
+	@return var(--b-#{$block}-#{$item});
+}
+
+@mixin set-alias($alias, $items) {
```

Comment:
Are the `set-X` mixin's needed now with the way `@use` works?

Reply:
nope, I must have copied those back in...